lte/gateway/c/core/oai/tasks/mme_app/mme_app_timer.hpp

```diff
@@ -20,10 +20,17 @@ extern "C" {
 
 #include "lte/gateway/c/core/oai/lib/itti/intertask_interface.h"
 #include "lte/gateway/c/core/oai/include/mme_app_ue_context.h"
```

Comment:
Separate PR is fine, but it will good if we could do before porting hash table to protobuf map. Because on inclusion of c++ provided features we get lot of errors as some piece of code is in 'C'